lib/p2p/Pool.ts

```diff
@@ -580,7 +583,16 @@ class Pool extends EventEmitter {
     peer.sendPacket(new packets.GetOrdersPacket({ pairIds: this.handshakeData.pairs }));
     if (this.config.discover) {
       // request peer's known nodes only if p2p.discover option is true
-      peer.sendPacket(new packets.GetNodesPacket());
+      if (this.config.discoverminutes === 0) {
+        // timer is disabled
+        peer.sendPacket(new packets.GetNodesPacket());
+        peer.getNodesTimer = undefined; // defensive programming
+      }else {
+        // timer is enabled
+        peer.getNodesTimer = setInterval(() => {
```

Comment:
looks like that if timer is enabled, there won't be an initial discovery - only after the interval.